pkg/minikube/tests/driver_mock.go

```diff
@@ -91,6 +97,10 @@ func (d *MockDriver) GetSSHKeyPath() string {
 // GetState returns the state of the driver
 func (d *MockDriver) GetState() (state.State, error) {
 	d.Logf("MockDriver.GetState: %v", d.CurrentState)
+	if d.NotExistError {
+		d.CurrentState = state.Error
+		return d.CurrentState, errors.New("machine does not exist")
```

Comment:
return error type instead of string.

Reply:
I added new error ErrorMachineNotExist but I didn't use it here.
I avoid import cycle because package `cluster` & `tests` depend on each other.
I added that comment.

FYI) `cluster` → `tests` → `cluster`